en/docs/learn/forced-password-reset.md

```diff
@@ -31,6 +31,12 @@ for forced password reset:
         on how to do this, see [Customizing Automated
         Emails](../../learn/customizing-automated-emails).
     
+2. Add the following configureation to the `<IS_HOME>/repository/conf/deployment.toml` file.
```

Comment:
Hi @sherenewso2
This is not a mandatory step, we can enable this in UI as well.

This is needed if we want to enable this feature for all the tenants by default.